.gitignore

```diff
@@ -1,19 +1,22 @@
-# Compiled python code
-*.pyc
-
-# osx files
-.DS_Store
-
-# sass cache
-.sass-cache
-.elasticbeanstalk
-
-# Distribution
-dist
-django_raster.egg-info
-
-# Docs build
-docs/_build
-
-# Coverage output
-.coverage
+# Compiled python code
+*.pyc
+
+# osx files
+.DS_Store
+
+# sass cache
+.sass-cache
+.elasticbeanstalk
+
+# Distribution
+dist
+django_raster.egg-info
+
+# Docs build
+docs/_build
+
+# Coverage output
+.coverage
```

Comment:
Makes sense to add PyCharm ignores here, did not know that .idea is a pycharm thing.